themes/bootstrap3/templates/checkouts/history.phtml

```diff
@@ -47,7 +47,7 @@
           <?php if ($purgeSelectedAllowed): ?>
             <label>
               <input type="checkbox" name="selectAll" class="checkbox-select-all"/>
-              <?=$this->transEsc('select_page')?>
+              <?=$this->transEsc('select_all_on_page')?>
             </label>
```

Comment:
It looks like the .checkbox on line 46 can be closed here instead of line... actually it may not be closed...

Reply:
The checkbox div is closed, but it is wrapped around not just the checkbox but also all of the buttons. This is consistent with the markup style in holds/list.phtml, myresearch/checkedout.phtml, myresearch/illrequests.phtml and myresearch/storageretrievalrequests.phtml as well. As a result, buttons do not wrap as a group, which is inconsistent with the desired behavior we're working on elsewhere. However, because these templates have comparatively few buttons, it is less likely to cause problems in these contexts. It is probably worth making our checkbox+button markup consistent everywhere, but I think fixing this immediately is less critical.